vue3-ts-quasar/src/composables/github/useGists.ts

```diff
@@ -0,0 +1,21 @@
+import { Ref } from 'vue';
+import { parseQuery } from '@/components/GistsPanel/parseQuery';
```

Comment:
Structurally, would recommend that we don't import dependencies from within components into composables.

Composables are meant to be reusable pieces of functionality within the application and thus adding them here would be coupling the composable to a GistsPanel dependency. This means if we wanted to use the get gists functionality with a different component/page, we'd need to pull the `parseQuery` functionality from the `GistsPanel` component.

## Suggestion
Migrate the `parseQuery` functionality into useGists. That way, it is self-contained.